app/javascript/crayons/CTAs/CTA.jsx

```diff
@@ -1,22 +1,47 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { Link } from '@crayons';
+import { defaultChildrenPropTypes } from '../../common-prop-types/default-children-prop-types';
+import { Icon } from '@crayons';
 
 export const CTA = (props) => {
-  const { variant = 'default', className, ...otherProps } = props;
+  const {
+    children,
+    href = '#',
+    variant,
+    icon,
+    className,
+    ...otherProps
+  } = props;
 
   const classes = classNames('c-cta', {
     [`c-cta--${variant}`]: variant,
+    'c-cta--icon-left': icon && children,
     [className]: className,
   });
 
-  return <Link block className={classes} {...otherProps} />;
+  return (
+    <a href={href} className={classes} {...otherProps}>
+      {icon && (
+        <Icon
+          src={icon}
+          aria-hidden="true"
+          focusable="false"
+          className={classNames('c-cta__icon')}
+        />
+      )}
+      {children}
+    </a>
+  );
 };
 
 CTA.displayName = 'CTA';
 
 CTA.propTypes = {
   variant: PropTypes.oneOf(['default', 'branded']),
+  rounded: PropTypes.bool,
+  href: PropTypes.string.isRequired,
   className: PropTypes.string,
+  children: defaultChildrenPropTypes.isRequired,
+  icon: PropTypes.ReactNode,
```

Comment:
I commented further down on the search section, but I think this also needs to be
```suggestion
  icon: PropTypes.elementType,
```